correios/models/data.py

```diff
@@ -635,6 +635,12 @@
         "Entrega prevista para o próximo dia útil",
         "Acompanhar",
     ),
+    ("FC", 47): (
+        "shipped",
```

Comment:
shouldn't be `delivery_rejected` ?

Reply:
+1

Reply:
Take a look at the above two statuses. As far as I understood the delivery was not attempted yet, this status could happen at any moment and we need to keep tracking it, so no.